community/modules/scripts/ramble-setup/templates/ramble_setup.yml.tpl

```diff
@@ -55,6 +63,12 @@
       recurse: true
     when: chmod_mode != "" and lock_out.rc == 0
 
+  - name: check if ramble profile exists
+    ansible.builtin.stat:
+      path: /etc/profile.d/ramble.sh
+    register: profile_check
+
   - name: Add ramble to profile
```

Comment:
I think the correct way to do this would by with:

```
  - name: Creating a file with content
    copy:
      dest: "/etc/profile.d/ramble.sh"
      content: ". {{ install_dir }}/share/ramble/setup-env.sh"
```

I think you could leave the stat check in place.